pkg/importer/dataStream.go

```diff
@@ -292,9 +403,6 @@ func (d *DataStream) pollProgress(reader *util.CountingReader, idleTime, pollInt
 func CopyData(dso *DataStreamOptions) error {
 	glog.V(1).Infof("copying %q to %q...\n", dso.Endpoint, dso.Dest)
 	switch dso.Source {
-	case controller.SourceRegistry:
-		glog.V(1).Infof("using skopeo to copy from registry")
-		return image.CopyRegistryImage(dso.Endpoint, dso.Dest, dso.AccessKey, dso.SecKey)
 	default:
```

Comment:
Switch statement with only default case

Reply:
Michael has a good point, at this point there no reason to have switch statement if we just have default and nothing else.